### ISSUE TITLE
[Nightly] Tapbacks don't send to really old chats

### ISSUE THREAD

**Issue by CSab6482** [BlueBubblesApp/BlueBubbles-Android-App#592]
I tried to send a tapback to a chat that hadn't had a message sent in it for about four months but it wouldn't go through and it stopped me from being able to send tapbacks to other chats

**Comment by zlshames** [comment]
@CSab6482 should i close this

**Comment by CSab6482** [comment]
> @CSab6482 should i close this

No, I just attempted to send a tapback to a group chat from April 11, 2021, and it didn't go through. The terminal doesn't give much information.
![122311444-30b38680-cee0-11eb-8db3-219664213380](https://user-images.githubusercontent.com/53708281/122311517-60628e80-cee0-11eb-8ce0-d0fd846e5ac0.png)
The disconnect happened because I clicked a notification from another app.
